src/DnDList.elm

```diff
@@ -813,3 +836,22 @@ type alias Position =
     { x : Float
     , y : Float
     }
+
+
+getCurrentHalf : Position -> Maybe Browser.Dom.Element -> ElementHalf
+getCurrentHalf position dropElement =
+    case dropElement of
+        Nothing ->
+            -- Doesn't matter
+            LeftHalf
+
+        Just element ->
+            let
+                halfX =
+                    element.element.x + element.element.width / 2
+            in
+            if position.x < halfX then
+                LeftHalf
+
+            else
+                RightHalf
```

Comment:
in the end the calculations aren't as bad!

Reply:
Using `InsertAround` in case of `OnDrop` works fine.
Do you have an idea, how do you want to define the behavior in case of `OnDrag`?

Reply:
I've just delegated to `stateUpdate InsertBefore` and `stateUpdate InsertAfter` in the end.

Reply:
`InsertAround + OnDrag` became `Rotate + OnDrag` now :)

Reply:
I don't understand :thinking:

Reply:
They are almost the same, but now I see that you added a TODO list, so this will be checked now:

![around-rotate](https://user-images.githubusercontent.com/8246853/89376652-9051da80-d6f0-11ea-9744-33560019cb0c.gif)

Reply:
Ooooh, are you kidding me, so Rotate does what I wanted? :facepalm:  I'll have to check this myself. Thanks for the GIF explanation!